docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -52,32 +53,45 @@ include::_attributes.adoc[] <3>
 +
 <1> Set the `id` value to be the same as the file name but without the extension. You can shorten this if the file name is too long.
 <2> For information about how to create a good title for each content type, see xref:{doc-guides}/doc-reference.adoc#titles-and-headings[Titles and headings] on the "Quarkus style and content guidelines" page.
-<3> The `_attributes.adoc` include is required to ensure that attributes get resolved, the table of contents is generated, and content renders in the website portal. 
+<3> The `_attributes.adoc` include is required to ensure that attributes get resolved, the table of contents is generated, and content renders in the website portal.
 <4> Set at least one category to ensure that the content is findable on the link:https://quarkus.io/guides/[Quarkus documentation home page]. For a list of Quarkus categories, see xref:{doc-guides}/doc-reference.adoc#document-attributes-and-variables[Document attributes and variables] on the "Quarkus style and content guidelines" page.
 +
 [IMPORTANT]
 ====
 Ensure there are no line breaks in the header section until after `:categories:` line.
 ====
+
+. Add an abstract that describes the purpose of the guide.
 +
-. Add an abstract to describe the purpose of the guide.
 [IMPORTANT]
 ====
-The first sentence of the abstract must explain the value and some benefit of the content in less than 27 words because this automatically displays on the link:https://quarkus.io/guides/[Quarkus guides homepage]. 
+The first sentence of the abstract must explain the value and some benefit of the content in less than 27 words because this automatically displays on the link:https://quarkus.io/guides/[Quarkus guides homepage].
 There must also be a line break before and after the abstract.
 ====
 
+. For how-to and tutorial topics, include a set of prerequisites. For example:
++
+[source,asciidoc]
+----
+.Prerequisites
+
+:prerequisites-time: 30 minutes <1>
+include::{includes}/prerequisites.adoc[] <2>
+----
+<1> Optional: Specifies the type of prerequisites to display, or sets a value that appears in the prerequisites. For more information about these attributes, inspect the content of the `docs/src/main/asciidoc/_includes/prerequisites.adoc` file.
```

Comment:
Re line 81: The way the numbering of the callouts render in the document, I initially thought it was a sub-procedure as I had not noticed the corresponding number in the code snippet itself. I wonder would it be better to present this info in a slightly different way to better distinguish it from the procedure, for example, as a bulleted list under the snippet. For example, 

- **prerequisites-time: (Optional): Specifies the type of .......
-  **include**: Includes the prerequisites..... 
Just a suggestion and I know there are also other callouts in the doc.

Reply:
Updated. Thanks.

Reply:
Updated. Thank you.